pypesto/store/read_from_hdf5.py

```diff
@@ -1,8 +1,38 @@
 import h5py
 from ..result import Result
 from ..optimize.result import OptimizerResult
+from ..profile.result import ProfilerResult
+from ..sample.result import McmcPtResult
 from ..problem import Problem
 from ..objective import Objective, ObjectiveBase
+import numpy as np
+
+
+def read_hdf5_profile(f: h5py.File,
+                      profile_id: h5py.Group,
+                      parameter_id: h5py.Group) -> 'ProfilerResult':
```

Comment:
Seem unintuitive that an ID would be of type h5py.Group

Reply:
To me this was the most explanatory name, since `profile_id` specifies the profile start and `parameter_id` specifies the parameter index. But i am open to change the names if you suggest something :)

Reply:
I checked again and you are right, `profile_id` is not a h5py.Group but a str, changed that for `profile_id`, `parameter_id` as wellt as `opt_id` and the function below.